### PR TITLE
Add ReadComicsOnline Presence

### LINKED ISSUE #4143: ReadComicsOnline | readcomiconline.li
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**Sape#1733** 

**https://readcomiconline.li/ | ReadComicsOnline** 

**The presence should include the name of the comic, the issue number, and the page number. When looking for a specific comic, the presence should show what you are searching for. Also, if the presence is idle on the home page or if someone so happens to be scrolling, make it say "Looking for something to read" in similarity to the youtube presence. When you select a comic, there is a page you are sent to before you the actual comic which is the page where it shows all the issues of said comic. Make it say (for example) "Star Wars: Darth Vader: Selecting issue" something along the lines. If there's anything that i said that isn't clear enough, reach out to my discord as I am in the premid server.** 

**If possible, please provide a logo for the service (512x512 minimum)**             
readcomicsonlineicon

### PR DESCRIPTION
**Pull request by vahya**
Closes #4143 

Screenshot 2021-07-24 202230
Screenshot 2021-07-24 202259
Screenshot 2021-07-24 202324
Screenshot 2021-07-24 202354
Screenshot 2021-07-24 202445
Screenshot 2021-07-24 202512
Screenshot 2021-07-24 202530

### CODE REVIEW

File: websites/R/ReadComicsOnline/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/rOrdlzw.png",
+  "thumbnail": "https://i.imgur.com/EuVcqxz.png",
+  "version": "1.0.0",
+  "color": "#6a0722",
+  "tags": ["comics", "online-reader"],
+  "url": "readcomiconline.li",
+  "description": {
+    "en": "ReadComicsOnline - Read High Quality Comics Online"
```
Comment (Bas950) [inline websites/R/ReadComicsOnline/dist/metadata.json:16]:
```suggestion
    "en": "Read comics online in high quality. Free download high quality comics."
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Please make sure the cookies meet our requirements and explain why they are needed and how they kinda work lmao

**Comment by vahya** [comment]
>Please make sure the cookies meet our requirements

Keys prefixed with ``PMD_`` 👍 

>explain why they are needed and how they kinda work

The site doesn't show what the user searched for when showing results, so whatever the user is searching for is stored in a cookie beforehand so it can be shown in the presence afterwards, when the results are shown on the website.

File: websites/R/ReadComicsOnline/presence.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          }
+        } else {
+          const title = document.querySelector<HTMLAnchorElement>("a.bigChar");
+          if (title) presenceData.details = title.innerText;
+          presenceData.state = "Looking for issue to read";
+          if (buttons) {
+            presenceData.buttons = [
+              {
+                label: "Check Description",
+                url: location.href
```
Comment (SlowLife1661) [inline websites/R/ReadComicsOnline/presence.ts:156]: Why is the one above "Read" but the one buttom "Check" xd

**Comment by SlowLife1661** [comment]
As for cookies fine by me of maybe add some sort of setting

**Comment by Bas950** [comment]
> As for cookies fine by me or maybe add some sort of setting

yea fine by me too, just fix the error mentioned above

**Comment by vahya** [comment]
I've added settings for disabling cookies.

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**